docs/application/native/guides/internationalization/i18n.md

```diff
@@ -773,6 +777,102 @@ The `current()` and `next()` functions only check the current index against the
 
 The UCharIter functions return code unit values in the range of 0 - 0xffff. Before any functions operating on strings are called, the string must be set with the `i18n_uchar_iter_set_string()`, `i18n_uchar_iter_set_UTF16BE()`, or `i18n_uchar_iter_set_UTF8()`function.
 
+
+<a name="plural_format"></a>
+## Formating plural messages with PluralFormat
```

Comment:
Please change as:
Format plural message with PluralFormat

Note: we avoid using gerund "ing" in the heading.